quinn-proto/src/crypto/rustls.rs

```diff
@@ -302,11 +305,13 @@ impl crypto::ServerConfig for rustls::ServerConfig {
         dst_cid: &ConnectionId,
     ) -> Result<Keys, UnsupportedVersion> {
         let version = interpret_version(version)?;
+        // TODO: why is this unwrap safe?
```

Comment:
Is this safe, and if so, why?

Reply:
I don't see why it would be. I think we should handle it similarly to `initial_version` above.